README.md

```diff
@@ -83,19 +83,18 @@ By contributing to this project, you agree to abide by its terms.
 
     To cite package 'brms.mmrm' in publications use:
 
-      Landau WM, Kunzmann K, Sidi Y, Stock C (????). _brms.mmrm: Bayesian
-      MMRMs using 'brms'_. R package version 0.0.2.9003,
-      https://github.com/openpharma/brms.mmrm,
-      <https://openpharma.github.io/brms.mmrm/>.
+      Landau WM, Kunzmann K, Sidi Y, Stock C (2024). _brms.mmrm: Bayesian
+      MMRMs using 'brms'_. R package version 0.1.0,
+      <https://CRAN.R-project.org/package=brms.mmrm>.
 
     A BibTeX entry for LaTeX users is
 
       @Manual{,
         title = {brms.mmrm: Bayesian MMRMs using 'brms'},
         author = {William Michael Landau and Kevin Kunzmann and Yoni Sidi and Christian Stock},
-        note = {R package version 0.0.2.9003, 
-    https://github.com/openpharma/brms.mmrm},
-        url = {https://openpharma.github.io/brms.mmrm/},
+        year = {2024},
+        note = {R package version 0.1.0},
+        url = {https://CRAN.R-project.org/package=brms.mmrm},
```

Comment:
Could we reference https://github.com/openpharma/brms.mmrm here?

Reply:
Addressed in https://github.com/openpharma/brms.mmrm/pull/89/commits/86a3ee0c7e4bde484609b0982673b70c449160c9.